e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/add-payment.spec.js

```diff
@@ -185,7 +186,7 @@ context('PDC_RECONCILE users can add a payment to a report', () => {
     cy.keyboardInput(cy.getInputByLabelText('Year'), '2023');
     cy.getInputByLabelText('Yes').click();
 
-    cy.contains('button', 'Continue').click();
+    continueButton().click();
```

Comment:
Sorry @RLavender98 should have mentioned - we can just use this command here and avoid the import 😎 

```js
cy.clickContinueButton();
```